STYLE.md

```diff
@@ -233,8 +233,8 @@ environment. In general, there should be no non-local network access. In additio
 
 Tests should be deterministic. They should not rely on randomness or details
 such as the current time. Instead, mocks such as
-[`MockRandomGenerator`](test/mocks/runtime/mocks.h) and
-[`Mock*TimeSource`](test/mocks/common.h) should be used.
+[`MockRandomGenerator`](test/mocks/common.h) and
+[`SimulatedTimeSystem`](test/test_common/simulated_time_system.h) should be used.
```

Comment:
nit: maybe `SimulatedTimeSystem is recommended` ?
I know real test time is still used in some places where inject-time mechanism is not provided. e.g., gRPC

Reply:
`should` implies it is recommended.
I'm assuming some randomness cannot be removed completely (from used libraries for example), but this should be the guideline for most tests.

Reply:
SGTM